package.json

```
@@ -1,17 +1,19 @@
 {
   "name": "@codexteam/icons",
   "version": "0.0.0",
```

if it works correctly we can bump version to stable 1.0.0

```suggestion
  "version": "1.0.0",
```